apps/cli/commands/init.py

```
@@ -0,0 +1,150 @@
+
+        # 4. Create PostgreSQL schema
+        console.print("[yellow]Creating PostgreSQL schema...[/yellow]")
+        create_postgresql_schema()
+        console.print("[green]✓ PostgreSQL schema created[/green]")
+
+        console.print("[bold green]✅ System initialized successfully![/bold green]")
+
+    except typer.Exit:
+        # Re-raise typer.Exit to preserve exit code
+        raise
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)